modules/bs.string/data/bs.string/function/find/find.mcfunction

```diff
@@ -0,0 +1,35 @@
+# ------------------------------------------------------------------------------------------------------------
+# Copyright (c) 2025 Gunivers
+#
+# This file is part of the Bookshelf project (https://github.com/mcbookshelf/Bookshelf).
+#
+# This source code is subject to the terms of the Mozilla Public License, v. 2.0.
+# If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Conditions:
+# - You may use this file in compliance with the MPL v2.0
+# - Any modifications must be documented and disclosed under the same license
+#
+# For more details, refer to the MPL v2.0.
+# ------------------------------------------------------------------------------------------------------------
+
+$scoreboard players set #bs.string.occurrence bs.data $(occurrence)
```

Comment:
no need to use bs.data can be bs.ctx if the function does not "leak" into user territory (with a callback) or with other functions from other modules